src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="/"
+    method="get"
+    class="search-bar"
+    data-qa="big">
```

Comment:
```suggestion
    <form
      action="/"
      method="get"
      class="search-bar"
      data-qa="big"
    >
```
если у тега больше двух атрибутов - необходимо размещать каждый с новой строки с отступами в два символа
закрывающая скобка должна быть на новой строке

 пофикси для всех тегов